packages/core/src/Stores/client-store.js

```diff
@@ -1838,8 +1838,6 @@ export default class ClientStore extends BaseStore {
         // broadcastAccountChange is already called after new connection is authorized
         if (!should_switch_socket_connection) this.broadcastAccountChange();
 
-        if (!this.is_virtual) this.getLimits();
```

Comment:
@wojciech-deriv Removed `get_limits` calling from `switchAccountHandler` in client store because it calls init() where we are calling `get_limits` again